components/actiongroup/index.css

```diff
@@ -79,12 +78,20 @@ governing permissions and limitations under the License.
       border-radius: 0;
       z-index: 0;
 
+      &:after {
+        --spectrum-actionbutton-focus-ring-border-radius-adjusted: 0;
+        test: var(--spectrum-actionbutton-focus-ring-border-radius-adjusted);
+      }
+
       &:first-child {
         border-start-start-radius: var(--spectrum-actionbutton-m-texticon-border-radius);
         border-end-start-radius: var(--spectrum-actionbutton-m-texticon-border-radius);
         margin-inline-end: calc(
           -1 * var(--spectrum-actionbutton-m-texticon-border-size) / 2
         );
+        &:after {
+          --spectrum-actionbutton-focus-ring-border-radius-adjusted: calc(var(--spectrum-actionbutton-m-quiet-textonly-border-radius) + var(--spectrum-actionbutton-m-textonly-focus-ring-gap)) 0 0 calc(var(--spectrum-actionbutton-m-quiet-textonly-border-radius) + var(--spectrum-actionbutton-m-textonly-focus-ring-gap));
```

Comment:
Does design want `0` or something tiny like 2px that would show slightly on retina displays?

0:
![image](https://user-images.githubusercontent.com/201344/151410177-d08af918-833a-44b9-ab31-d05726bab90a.png)


1px:
![image](https://user-images.githubusercontent.com/201344/151410351-c944a153-0e21-4dd7-9b5e-56812268af61.png)